composer/utils/batch_helpers.py

```diff
@@ -0,0 +1,99 @@
+from typing import Any
+
+from composer.core.types import BatchKeys
+
+
+def batch_get(batch: Any, key: BatchKeys) -> Any:
```

Comment:
I would be OK with typing the `key` as `Any` -- main reason being that if if the batch is a dict, we do not know what type the key would be -- it's likely to be a string, but could really be anything.

Reply:
yes, agreed!

Reply:
ok fixed in  [b64b46c](https://github.com/mosaicml/composer/pull/978/commits/b64b46ce1559530b6ccb8dd3747d5799f583d727) and 
d9c6694